core-plugins/src/main/java/co/cask/hydrator/plugin/batch/aggregator/function/CollectList.java

```diff
@@ -0,0 +1,44 @@
+package co.cask.hydrator.plugin.batch.aggregator.function;
+
+import co.cask.cdap.api.data.format.StructuredRecord;
+import co.cask.cdap.api.data.schema.Schema;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Collect List of a specific column
+ * @param <T> type of aggregate value
+ */
+public class CollectList<T> implements AggregateFunction<List<T>> {
+
+    private List<T> collectList;
```

Comment:
use 2 space indentation for consistency with other code

Reply:
I'll make the necessary changes

Reply:
updated